app/components/views/GetStartedPage/SetupWallet/SetupWallet.jsx

```diff
@@ -0,0 +1,199 @@
+import { injectIntl } from "react-intl";
+import { withRouter } from "react-router";
+import { createElement as h } from "react";
+import { useState, useEffect, useCallback } from "react";
+import { useService } from "@xstate/react";
+import SetupWalletPage from "./SetupWalletPage";
+import SettingMixedAccount from "./SetMixedAcctPage/SetMixedAcctPage";
+import ProcessUnmanagedTickets from "./ProcessUnmanagedTickets/ProcessUnmanagedTickets";
+import ProcessManagedTickets from "./ProcessManagedTickets/ProcessManagedTickets";
+import { useDaemonStartup, useAccounts, usePrevious } from "hooks";
+import { IMMATURE, LIVE, UNMINED } from "constants/Decrediton";
+import { FormattedMessage as T } from "react-intl";
+
+const SetupWallet = ({ settingUpWalletRef }) => {
+  const {
+    getCoinjoinOutputspByAcct,
+    stakeTransactions,
+    onProcessManagedTickets,
+    goToHome,
+    onProcessUnmanagedTickets,
+    isProcessingUnmanaged,
+    isProcessingManaged
+  } = useDaemonStartup();
+  const { mixedAccount } = useAccounts();
```

Comment:
i would create another custom hook ie `useSetupWallet` and would move all state logic (including all useCallback wrapped functions) there too cleanup up this `.jsx` and improve it's readability.